src/App.jsx

```diff
@@ -1,248 +1,303 @@
+  const sortByAnna = () => {
+    setSelected('Anna')
+  }
+
+  const sortByMax = () => {
+    setSelected('Max')
+  }
+
+  const sortByJohn = () => {
+    setSelected('John')
+  }
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title">Product Categories</h1>
+
+        <div className="block">
+          <nav className="panel">
+            <p className="panel-heading">Filters</p>
+
+            <p className="panel-tabs has-text-weight-bold">
+              <a
+                onClick={sortByAll}
+                className={select === 'All' ? "is-active" : ""}
+                data-cy="FilterAllUsers"
+                href="#/"
+              >
+                All
+              </a>
+
+              <a
+                onClick={sortByRoma}
+                className={select === 'Roma' ? "is-active" : ""}
+                data-cy="FilterUser"
+                href="#/"
+              >
+                Roma
+              </a>
+
+              <a
+                onClick={sortByAnna}
+                className={select === 'Anna' ? "is-active" : ""}
+                data-cy="FilterUser"
+                href="#/"
               >
                 Anna
-              </td>
-            </tr>
+              </a>
 
-            <tr data-cy="Product">
-              <td className="has-text-weight-bold" data-cy="ProductId">
-                3
-              </td>
+              <a
+                onClick={sortByMax}
+                className={select === 'Max' ? "is-active" : ""}
+                data-cy="FilterUser"
+                href="#/"
+              >
+                Max
+              </a>
 
-              <td data-cy="ProductName">iPhone</td>
-              <td data-cy="ProductCategory">💻 - Electronics</td>
+              <a
+                onClick={sortByJohn}
+                className={select === 'John' ? "is-active" : ""}
+                data-cy="FilterUser"
+                href="#/"
+              >
+                John
+              </a>
+            </p>
+
+            <div className="panel-block">
+              <SearchBlock deleteBtn={deleteBtn} setQuery={setQuery}/>
+            </div>
 
-              <td
-                data-cy="ProductUser"
-                className="has-text-link"
+            <div className="panel-block is-flex-wrap-wrap">
+              <a
+                href="#/"
+                data-cy="AllCategories"
+                className="button is-success mr-6 is-outlined"
               >
-                Roma
-              </td>
-            </tr>
-          </tbody>
-        </table>
+                All
+              </a>
+
+              <a
+              // is-info
+                onClick={sortByGrocery}
+                data-cy="Category"
+                className={select === 'Grocery' ? "button mr-2 my-1 is-info" : "button mr-2 my-1"}
+                href="#/"
+              >
+                Grocery
+              </a>
```

Comment:
You need to use `map` method for categories from server here instead of repeating each category